javalib/src/main/scala/java/nio/file/Files.scala

```diff
@@ -627,36 +628,56 @@ object Files {
 
       if (dirsToSkip.contains(parent)) ()
       else {
-        val attributes = getFileAttributeView(p,
-                                              classOf[BasicFileAttributeView],
-                                              Array.empty).readAttributes()
-        while (openDirs.nonEmpty && !parent.startsWith(openDirs.head)) {
-          visitor.postVisitDirectory(openDirs.pop(), null)
-        }
+        try {
+
+          // The sense of how LinkOption follows links or not is somewhat
```

Comment:
This is a good explanation. I would really like to know what they were thinking when they introduced `LinkOption.NOFOLLOW_LINKS` and `FileVisitOption.FOLLOW_LINKS` instead of having just one or the other option in each class.

Reply:
Thank you.